src/components/SocialIcons.vue

```diff
@@ -0,0 +1,25 @@
+<script setup lang="ts">
+defineProps<{
+  member: string
+  socials: Array<string>
```

Comment:
For convention across the project, this type should be `string[]`.

Reply:
[fix(components): align array type to project standard](https://github.com/Schrodinger-Hat/schrodinger-hat-website/pull/94/commits/13a889df677e94400d0b4f532691c98298cb316d)
fixed